README.md

```diff
@@ -1,13 +1,149 @@
-# connecthub
+# ConnectHub
 
-# Run the web version locally
+***
 
-This project is unlikely to be deployed, everything will happen locally. To run the website:
+## Overview
+ConnectHub is a forum for meaningful discussions, offering easy post management and personalized feeds based on your interests.
```

Comment:
think we should add 
  Says why the project was made (CSC207project)
 Gives the user a general sense of what problem this project solves and whether it is a useful project for them
(this project helps connect students/etc. together and was helpful for learning java LOL or something 💀 )

Reply:
yea whatever this is so corny lmao